bin/eslint.js

```diff
@@ -165,6 +165,10 @@ runWithTiming("engineConfig", function () {
       options.extensions = userConfig.extensions;
     }
 
+    if (userConfig.ignorePath) {
+      options.ignorePath = userConfig.ignorePath;
```

Comment:
Can you read this as `ignore_path` out of `userConfig`? Since this value is what the user has in their YAML file, I think we want to keep the underscore convention there.